source/config.js

```diff
@@ -4,11 +4,7 @@
  * Configuration
  */
 
-var secure = window.location.protocol === 'https:';
-
 module.exports = {
     name: 'likely',
-    prefix: 'likely__',
-    secure: secure,
-    protocol: secure ? 'https:' : 'http:',
+    prefix: 'likely__'
```

Comment:
Yep, except that I’d rather name them `check-codestyle` and `fix-codestyle` to make them sound like a proper phrase.